api/candlepin-api-spec.yaml

```diff
@@ -6687,6 +8216,23 @@ components:
         href:
           type: string
 
+    DateRange:
+      description: Represents a range in time, start date to end Date inclusive
+      properties:
+        startDate:
+          type: string
+          format: date-time
+        endDate:
+          type: string
+          format: date-time
+
+    DeleteResult:
```

Comment:
Need to delete `org.candlepin.dto.api.v1.DeleteResult` as this needs to be used.

Reply:
Done.